src/test/scala/integration/TopicLoaderIntSpec.scala

```diff
@@ -214,13 +214,18 @@ class TopicLoaderIntSpec extends KafkaSpecBase[IO] {
   }
 
   private trait TestContext[F[_]] {
-    implicit val kafkaConfig: EmbeddedKafkaConfig =
-      EmbeddedKafkaConfig(kafkaPort = RandomPort(), zooKeeperPort = RandomPort(), Map("log.roll.ms" -> "10"))
+    implicit val kafkaConfig: EmbeddedKafkaConfig
```

Comment:
```suggestion
    given kafkaConfig: EmbeddedKafkaConfig
```

Reply:
Atm we're cross compiling Scala 2.13 and 3, so we need to use implicits etc. Raised https://github.com/sky-uk/fs2-kafka-topic-loader/issues/9 to discuss dropping Scala 2.13 support as I'm not sure it's necessary.

Reply:
https://github.com/sky-uk/fs2-kafka-topic-loader/blob/9e60e7e924735ebe4c4532f2422b7d556e18eecf/build.sbt#L5